res/values/strings.xml

```diff
@@ -364,5 +364,14 @@
     <string name="file_list__footer__files">%1$d files</string>
     <string name="file_list__footer__files_and_folder">%1$d files, 1 folder</string>
     <string name="file_list__footer__files_and_folders">%1$d files, %2$d folders</string>
+    <string name="prefs_instant_behaviour_dialogTitle">Upload file to server and ...</string>
+    <string name="prefs_instant_behaviour_title">Behaviour</string>
+    <string name="upload_copy_files">Copy file</string>
+    <string name="upload_move_files">Move file</string>
+
+    <string name="pref_behaviour_entries_do_nothing">do nothing</string>
+    <string name="pref_behaviour_entries_copy">copy file to OC folder</string>
+    <string name="pref_behaviour_entries_move">move file to OC folder</string>
```

Comment:
Maybe the easiest solution is to define the strings as:
<string name="pref_behaviour_entries_do_nothing">Do nothing</string>
<string name="pref_behaviour_entries_copy">Copy file to app folder</string>
<string name="pref_behaviour_entries_move">Move file to app folder</string>

We avoid to put the appname in the strings, because if we change the appname we avoid to review every appname reference in the xml files.